toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -2159,6 +2170,33 @@ func runPostInstallScripts(installChroot *safechroot.Chroot, config configuratio
 	return
 }
 
+func cleanupTdnfCache(installChroot *safechroot.Chroot) error {
+	const (
+		squashErrors      = false
+		rpmCacheDirectory = "/var/cache/tdnf"
+	)
+
+	ReportActionf("Cleaning tdnf cache")
+	err := installChroot.UnsafeRun(func() error {
+		chrootErr := shell.ExecuteLive(squashErrors, "tdnf", "clean", "all")
+		return chrootErr
+	})
+
+	if err != nil {
+		err = fmt.Errorf("failed to cleanup tdnf cache:\n%w", err)
+		return err
+	}
+
+	cacheDir := filepath.Join(installChroot.RootDir(), rpmCacheDirectory)
+	err = os.RemoveAll(cacheDir)
+	if err != nil {
+		err = fmt.Errorf("failed to remove tdnf cache directory (%s):\n%w", cacheDir, err)
+		return err
+	}
+
```

Comment:
Is that really needed? 
I am not in favor of removing dirs as in the past that led to SELinux labels being messed up on boot.

Reply:
I believe the right way to clean tdnf cache is to remove `/var/cache/tdnf`. But alternatively we could just remove everything in that folder itself

Reply:
fwiw this is a direct copy of what we are doing today via script. Won't claim one way or the other which is correct.

Reply:
Yeah, I can confirm that it is necessary to at *least* delete everything in `/var/cache/tdnf`, I worked added this to our core images scripts and found that just running `tdnf -y clean all` was not sufficient.

Reply:
can we remove files under /var/cache/tdnf instead? 
I don't believe any of the current images that uses that have SELinux on.